R/downloadableReactTable.R

```diff
@@ -0,0 +1,166 @@
+# -------------------------------------------------
+# -- Application Downloadable React Table Module --
+# -------------------------------------------------
+
+
+#' downloadableReactTable module UI function
+#'
+#' downloadableReactTable module is extending \code{?reactable} package table functions by creating
+#' a custom high-functionality table paired with \link[periscope2]{downloadFile} button.
+#' The table has search and highlight functionality, infinite scrolling, sorting by columns and
```

Comment:
maybe changing infinite scrolling -> pagination would be more aligned with the reactable terminology

Reply:
by default pagination is off, so I think leave this as infinite scrolling.  But we need to say "The table has the following default functionality: search, highlight, infinite scrolling, sorting by ... " etc.  Because it is just by default, the user can change all of these.